pkg/operator/profilecalculator.go

```diff
@@ -158,7 +158,9 @@ func (pc *ProfileCalculator) nodeChangeHandler(nodeName string) (bool, error) {
 // * whether to run the Tuned daemon in debug mode on node nodeName
 // * an error if any
 func (pc *ProfileCalculator) calculateProfile(nodeName string) (string, map[string]string, tunedv1.OperandConfig, error) {
-	var operand tunedv1.OperandConfig
+	var (
+		operand tunedv1.OperandConfig
+	)
```

Comment:
why do we need this change?

Reply:
We do not. :) I think it is an artifact of me messing with the code, adding variables, then removing them and forgetting to rollback to the original.

Reply:
@ffromani Changed this back to one line.